lms/tasks/email_digests.py

```diff
@@ -104,7 +105,6 @@ def send_instructor_email_digest_tasks(*, batch_size):  # pylint:disable=unused-
                     (),
                     {
                         "h_userid": h_userid,
-                        "created_after": created_after.isoformat(),
```

Comment:
This is the root `send_instructor_email_digest_tasks()` task that runs once a day and queues many instances of the `send_instructor_email_digest()` task below.

Previously all emails covered a 24hr period so it made sense for the root task to calculate `created_after` once here and pass the same `created_after` value to every child task that it queues.

Now each email covers a different time period (up to a week) depending on when we last emailed each user according to the `task_done` table, so there's no longer a single `created_after` value for each instance of the child task, it's different for each instance.

We don't want this root task to have to compute the `created_after`'s for each user because then this root task would have to do thousands of `task_done` queries (or one huge query) in a single task instance.

So `created_after` is just removed from this root task entirely.